package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "javascript-express-api-postgres",
   "version": "1.0.0",
-  "description": "",
+  "description": "Assignments are located [here](./Assignments.md).",
```

Comment:
Yikes.  Can we revert `package.json` file?  We shouldn't make any changes to this file.